sdcm/cluster.py

```diff
@@ -1335,6 +1384,45 @@ def disable_client_encrypt(self):
         self.stop_scylla()
         self.start_scylla()
 
+    def prepare_files_for_archive(self, fileslist):
+        """Prepare files for creating archives on node
+
+        Create directories structure and copy files
+        from absolute path of files in fileslist
+        in node /tmp/node_name and return path to created
+        structure
+        Ex.:
+            filelists = ['/proc/cpuinfo', /var/log/system.log]
+
+            result directory:
+            /tmp/<node_name>/proc/cpuinfo
+                            /var/log/system.log
+
+        Arguments:
+            fileslist {list} -- list of file with fullpaths on node
+
+        Returns:
+            [type] -- path to created directory structure
+        """
+        root_dir = '/tmp/%s' % self.name
+        self.remoter.run('mkdir %s' % root_dir)
+        for f in fileslist:
+            if self.file_exists(f):
+                old_full_path = os.path.dirname(f)[1:]
+                new_full_path = os.path.join(root_dir, old_full_path)
+                self.remoter.run('mkdir -p %s' % new_full_path)
+                self.remoter.run('cp -r %s %s' % (f, new_full_path))
+        return root_dir
+
+    def get_console_output(self):
+        #TODO add GCE
```

Comment:
On `GCENode`

- Please create this method (don't implement)
- add Todo-s comment
- output warning to the log
- create a task in Trello in the pre-meetings column to add this functionality on GCE

Otherwise the code will fail in GCE during `archive_system_details_information` run.